<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>375</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>共产主义</string>
  </property>
  <property name="windowIcon">
   <iconset resource="music.qrc">
    <normaloff>:/image/app</normaloff>:/image/app</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>10</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTableWidget" name="listView">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>音乐标题</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>歌手</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>专辑</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>年份</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>时长</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="songLabel">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="music.qrc">:/image/defaultLabel</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="songInfo">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>96</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size:30px;font-family:Microsoft YaHei;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="previousButton">
              <property name="maximumSize">
               <size>
                <width>1111</width>
                <height>1111</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="music.qrc">
                <normaloff>:/image/previousimage</normaloff>:/image/previousimage</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="playButton">
              <property name="maximumSize">
               <size>
                <width>11111111</width>
                <height>11111</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">outline:none;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="music.qrc">
                <normaloff>:/image/playimage</normaloff>:/image/playimage</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="nextButton">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="music.qrc">
                <normaloff>:/image/nextimage</normaloff>:/image/nextimage</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="modelButton">
              <property name="maximumSize">
               <size>
                <width>18</width>
                <height>15</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="music.qrc">
                <normaloff>:/image/loop</normaloff>:/image/loop</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="volumnButton">
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="music.qrc">
                  <normaloff>:/image/smallvolumnimage</normaloff>:/image/smallvolumnimage</iconset>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="volumnBar">
                <property name="maximumSize">
                 <size>
                  <width>76</width>
                  <height>23</height>
                 </size>
                </property>
                <property name="value">
                 <number>30</number>
                </property>
                <property name="tracking">
                 <bool>true</bool>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="songProgress">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>10</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="timelabel">
          <property name="styleSheet">
           <string notr="true">QLabel{background-color:rgba(100,100,100,0.2);font-size:15px;}</string>
          </property>
          <property name="text">
           <string>0:00\0:00</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="playrateLabel">
          <property name="styleSheet">
           <string notr="true">QLabel{font-size:15px;}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>27</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="listButton">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="music.qrc">
            <normaloff>:/image/listDown</normaloff>:/image/listDown</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>659</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>播放</string>
    </property>
    <widget class="QMenu" name="menuPlaymode">
     <property name="title">
      <string>循环模式</string>
     </property>
     <addaction name="actionLoop"/>
     <addaction name="actionCurrentItemInLoop"/>
     <addaction name="actionSequential"/>
     <addaction name="actionRandom"/>
    </widget>
    <addaction name="actionPlay"/>
    <addaction name="actionPause"/>
    <addaction name="actionStop"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
    <addaction name="actionFaster"/>
    <addaction name="actionSlower"/>
    <addaction name="menuPlaymode"/>
    <addaction name="actionAddprogress"/>
    <addaction name="actionSubprograss"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>列表</string>
    </property>
    <addaction name="actionImport_local_files"/>
    <addaction name="actionClear_list"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>皮肤</string>
    </property>
    <addaction name="actionbgblue"/>
    <addaction name="actionbgyellow"/>
    <addaction name="actionbggreen"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionGong"/>
    <addaction name="action_Auther"/>
    <addaction name="action_Qt"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <action name="actionXinjian">
   <property name="text">
    <string>xinjian</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="music.qrc">
     <normaloff>:/image/playicon</normaloff>:/image/playicon</iconset>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="music.qrc">
     <normaloff>:/image/pauseicon</normaloff>:/image/pauseicon</iconset>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="music.qrc">
     <normaloff>:/image/stopicon</normaloff>:/image/stopicon</iconset>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionImport_local_files">
   <property name="text">
    <string>本地歌单导入</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="actionClear_list">
   <property name="text">
    <string>清空列表</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionbgblue">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>青鸟</string>
   </property>
  </action>
  <action name="actionbgyellow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>秋叶</string>
   </property>
  </action>
  <action name="actionbggreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>绿荫</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="music.qrc">
     <normaloff>:/image/nexticon</normaloff>:/image/nexticon</iconset>
   </property>
   <property name="text">
    <string>下一首</string>
   </property>
   <property name="shortcut">
    <string>.</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="music.qrc">
     <normaloff>:/image/previousicon</normaloff>:/image/previousicon</iconset>
   </property>
   <property name="text">
    <string>上一首</string>
   </property>
   <property name="shortcut">
    <string>,</string>
   </property>
  </action>
  <action name="actionFaster">
   <property name="icon">
    <iconset resource="music.qrc">
     <normaloff>:/image/fastericon</normaloff>:/image/fastericon</iconset>
   </property>
   <property name="text">
    <string>速度+</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionSlower">
   <property name="icon">
    <iconset resource="music.qrc">
     <normaloff>:/image/slowericon</normaloff>:/image/slowericon</iconset>
   </property>
   <property name="text">
    <string>速度-</string>
   </property>
   <property name="shortcut">
    <string>Z</string>
   </property>
  </action>
  <action name="actionLoop">
   <property name="text">
    <string>列表循环</string>
   </property>
  </action>
  <action name="actionCurrentItemInLoop">
   <property name="text">
    <string>单曲循环</string>
   </property>
  </action>
  <action name="actionSequential">
   <property name="text">
    <string>顺序播放</string>
   </property>
  </action>
  <action name="actionRandom">
   <property name="text">
    <string>随机播放</string>
   </property>
  </action>
  <action name="actionAddprogress">
   <property name="text">
    <string>进度+</string>
   </property>
  </action>
  <action name="actionSubprograss">
   <property name="text">
    <string>进度-</string>
   </property>
  </action>
  <action name="actionGong">
   <property name="text">
    <string>关于共产主义</string>
   </property>
  </action>
  <action name="action_Auther">
   <property name="text">
    <string>关于作者</string>
   </property>
  </action>
  <action name="action_Qt">
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="music.qrc"/>
 </resources>
 <connections/>
</ui>
